demo/site/src/middleware.ts

```diff
@@ -22,18 +24,7 @@ export default chain([
 ]);
 
 export const config = {
-    matcher: [
-        /*
-         * Match all request paths except for the ones starting with:
-         * - _next/static (static files)
-         * - _next/image (image optimization files)
-         * - favicon.ico, favicon.svg, favicon.png
-         * - manifest.json
-         * - assets (assets from /public folder)
-         * - robots.txt
-         */
-        "/((?!_next/static|_next/image|favicon.ico|favicon.svg|favicon.png|manifest.json|assets/|robots.txt).*)",
```

Comment:
having a closer look at this regex I see other issues:
- the `.` (in favicon.* and others) is not exacped, so it matches faviconXsvg
- it doesn not include `^` to match at start of string (I really hope some magic in next adds this though)
- (minor) missing `/` in _next/static wil also match _next/staticxxx, not just _next/static/xxx

Note that these issues are also in the patterns from the nextjs docs!